app/controllers/admin/bulk_renewals_controller.rb

```diff
@@ -0,0 +1,21 @@
+module Admin
+  class BulkRenewalsController < BaseController
+    include ActionView::RecordIdentifier
+
+    before_action :set_member, only: [:update]
+
+    def update
+      @member.loans.checked_out.each do |loan|
+        loan.renew! if loan.renewable?
+      end
```

Comment:
This probably isn't optimal if there is the case where a member has a very large number of items out on loan. However, I think this is a decent first iteration.   

Other thoughts include moving the singular or bulk renewal process out to a service object to avoid having that logic here.

Reply:
Good call-out. The most we ever have to renew at once is ~12 items, so I think this is OK to start with.